missing requirements.txt

I think you forgot to add a requirements.txt file for pip

These are the packages I had to install by trial and error to get `py.test -s -k test_deploy_simple` running: pytest, requests, fabric, jinja2, docker-py

That said I only got so far as to get an error with connecting to Docker, it seems docker-py is assuming it's running on Linux as it's trying to connect via Unix socket:

```
================ test session starts =================
platform darwin -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /Users/dain/workspace/docker_navitia, inifile:
collected 0 items / 1 errors

======================= ERRORS =======================
_____ ERROR collecting tests/test_deploy.py _____
tests/test_deploy.py:15: in <module>
    from docker_navitia.docker_navitia import ROOT, BuildDockerSimple, BuildDockerCompose, find_image, find_container
docker_navitia.py:24: in <module>
    docker_client = docker.Client(base_url='unix://var/run/docker.sock', version='auto')
/Library/Python/2.7/site-packages/docker/client.py:77: in __init__
    self._version = self._retrieve_server_version()
/Library/Python/2.7/site-packages/docker/client.py:97: in _retrieve_server_version
    'Error while fetching server API version: {0}'.format(e)
E   DockerException: Error while fetching server API version: ('Connection aborted.', error(2, 'No such file or directory'))
============== 1 error in 0.38 seconds ===========
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -2,3 +2,5 @@ docker-compose==1.2.0
 docker-py==1.1.0
 pytest==2.7.1
 requests==2.5.3
+fabric==1.10.2
+jinja2==2.8

--- a/requirements.txt
+++ b/requirements.txt
@@ -4,3 +4,4 @@ pytest==2.7.1
 requests==2.5.3
 fabric==1.10.2
 jinja2==2.8
+nose==1.3.7

